### ISSUE TITLE
cant generate openapi spec with class-validator@^0.12.1

### ISSUE THREAD

**Issue by amitgilad3** [epiphone/routing-controllers-openapi#44]
when upgrading to  "class-validator": "^0.12.1"  
const metadatas = (getFromContainer(MetadataStorage) as any).validationMetadatas
returns empty array 
you can reproduce  this by updating class-validator in the sample project .

**Comment by whereiswolf** [comment]
@amitgilad3 I found that you can change the code as below to make `routing-controllers-openapi` work the same as before. Seems to work just fine.
```diff
import { getMetadataArgsStorage } from 'routing-controllers'
import { routingControllersToSpec } from 'routing-controllers-openapi'
--- import { getFromContainer, MetadataStorage } from 'class-validator'
+++ import { getMetadataStorage } from 'class-validator'

--- const { validationMetadatas } = getFromContainer(MetadataStorage) as any
+++ const { validationMetadatas } = getMetadataStorage() as any
const schemas = validationMetadatasToSchemas(validationMetadatas, {
  refPointerPrefix: '#/components/schemas/',
})

const storage = getMetadataArgsStorage()
...
```
Hope it helps, cheers 😄!

**Comment by amitgilad3** [comment]
Thanks @whereiswolf  for the answer but like you wrote i encountered the bug you metioned .
this solution doesn't work :(

**Comment by epiphone** [comment]
I'm looking into this in https://github.com/epiphone/class-validator-jsonschema/issues/31. Thanks for reporting!

**Comment by epiphone** [comment]
Alright, I baked together a pre-release. Can you try `yarn add class-validator-jsonschema@next` & `yarn add routing-controllers-openapi@next` and let me know how it works?

**Comment by whereiswolf** [comment]
@epiphone It seems to work just fine 😄🎉 Thanks for the effort!

**Comment by epiphone** [comment]
Neat! Released in v2.0.0

**Comment by amitgilad3** [comment]
thanks for the awsome work!!
